types/unix-time/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare const unixTime: UnixTime
```

Comment:
when using scaffold code with templates for module, all the 'docu' like stuff is added (header file with version/author/reference).
There wouldn't be required to add all those excusions.